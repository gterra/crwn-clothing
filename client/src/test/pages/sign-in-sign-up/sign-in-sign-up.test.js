import React from "react";
import { shallow } from "enzyme";
import SignInSignUpPage from "../../../pages/sign-in-sign-up/sign-in-sign-up.component";

it("should render SignInSignUpPage component", () => {
  expect(shallow(<SignInSignUpPage />)).toMatchSnapshot();
});
