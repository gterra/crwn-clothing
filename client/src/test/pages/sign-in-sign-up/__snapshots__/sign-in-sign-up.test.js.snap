// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignInSignUpPage component 1`] = `
<styled.div>
  <Connect(SignIn) />
  <Connect(SignUp) />
</styled.div>
`;
